distribution/openrct2.d.ts

```diff
@@ -822,8 +822,7 @@ declare global {
         frameBase: number;
         frameCount?: number;
         frameDuration?: number;
-        offsetX?: number;
-        offsetY?: number;
+        offset?: Coord2;
```

Comment:
We should rename `Coord2` and `Coord3` to either the same as their counterparts in C++ or something more meaningful at some point

Reply:
At the moment they are used for both map and screen coordinates. There is no strict type checking in JavaScript and TypeScript will be happy if the interface has the same fields which they will do.

Maybe I will change it, but in a separate PR.

Reply:
Yeah, I'm ok with it being in a different PR (I thought it would be), just don't want it to be forgotten.